### ISSUE TITLE
Narwhal Bot isn't triggering tests when it comments "/test all"

### ISSUE THREAD

**Issue by RothAndrew** [defenseunicorns/delivery-aws-iac#353]
<img width="1255" alt="image" src="https://github.com/defenseunicorns/delivery-aws-iac/assets/16000938/68ecd4ef-cee8-4b82-bcc6-673b5af4cde1">

https://github.com/defenseunicorns/delivery-aws-iac/actions/runs/6184799976/job/16789130985?pr=352

**Comment by zack-is-cool** [comment]
yeah.. I think we may need to trigger it similarly to how we trigger update in that same workflow
